packages/ui-extensions/src/surfaces/point-of-sale/run/intent/BaseIntent.ts

```diff
@@ -0,0 +1,8 @@
+interface ExtensionError extends Error {
+  level: 'error' | 'warning';
```

Comment:
Wondering what the purpose of error vs warning will be on the POS side. Do we have a recommendation as to when either will be appropriate?

Reply:
https://docs.google.com/document/d/1km0Oyz_QZTWUdFtTWAv3-1qmVNsaD_shY4VAmKwSZ2g/edit?tab=t.0#bookmark=id.39h1v6blsjs5 This is to trigger the yellow or red states. We can expound on it in docs etc